### PR TITLE
Add standard GitHub labels

### PR DESCRIPTION
**Pull request by morgante**
This adds code to manage all GitHub labels in our repo using Terraform.

### CODE REVIEW

**Comment by aaron-lane** [comment]
FYI @onetwopunch

**Comment by aaron-lane** [comment]
@morgante My impression from our conversation on November 12 was that labels would be managed at the organization level rather than on each repository. What was the motivation to come back to this approach?

**Comment by morgante** [comment]
@aaron-lane It turns out the org labels only get added to new repos, not existing ones.

File: infra/terraform/test-org/github/versions.tf [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+terraform {
+  required_version = "~> 0.12.0"
+}
+
+provider "external" {
```
Comment (aaron-lane) [inline infra/terraform/test-org/github/versions.tf]: These providers appear to be unnecessary.

File: infra/terraform/test-org/github/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+## Operating
+
+This module requires setting a GitHub personal access token.
+
+First, create a [personal access token](https://help.github.com/en/enterprise/2.17/user/authenticating-to-github/creating-a-personal-access-token-for-the-command-line#creating-a-token).
+
+Then, export it as a Terraform variable:
+
+```
+export TF_VAR_github_token=aaaaaaa
```
Comment (aaron-lane) [inline infra/terraform/test-org/github/README.md]: I suggest we stick with our standard for configuring credentials and just require [GITHUB_TOKEN](https://www.terraform.io/docs/providers/github/#token) to be defined in the environment.

**Pull request merged**